tests/dummy/app/pods/demo/template.hbs

```diff
@@ -80,13 +80,13 @@
     {{/frost-button}}
   {{/frost-popover}}
 {{/frost-button}}
-<br />
+<br >
```

Comment:
Likely don't need the space before the closing bracket

Reply:
Fixed